### DISCUSSION TITLE
Развитие документации

### DISCUSSION THREAD

**Discussion by alkoleft** [bia-technologies/yaxunit#423] [Планы развития]
* Инструкции по внесению вклада
  * Код
  * Документация
  * Issue
* Инструменты контроля наличия документации
* Автогенерация доки с помощью VA
* Развитие генерации доки с помощью BSL-Doc
* Сбор практик подходов по различным вариантам тестирования
* Сбор вопросов (придумать как их собирать и агрегировать) и ответов

**Comment by alexandr-yang** [comment]
Как вариант, добавить версионирование документации. Не всегда понятно в какой версии тот или иной функционал был добавлен

**Reply by alkoleft** [reply]
Да нужно, есть задача на это, на днях попробовал. Думаю теперь как сделать для старых версий.
